<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwarmGUI</class>
 <widget class="QMainWindow" name="SwarmGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="takeoffButton">
       <property name="text">
        <string>Take off</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="landButton">
       <property name="text">
        <string>Land</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hoverButton">
       <property name="text">
        <string>Hover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Move on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="LiveScreen" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>701</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControll">
    <property name="title">
     <string>Control</string>
    </property>
   </widget>
   <addaction name="menuControll"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LiveScreen</class>
   <extends>QWidget</extends>
   <header location="global">livescreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SwarmGUI.qrc"/>
 </resources>
 <connections/>
</ui>
